0000-template.md

```diff
@@ -1,6 +1,7 @@
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
 - Target Major Version: (2.x / 3.x / X.x)
 - Reference Issues: (fill in existing related issues, if any)
+- Status: Pending (change to Active if the RFC is accepted and merged)
```

Comment:
Should we have more of a "timeline" section, which lists the start date, date the FCP started, date it became Active, etc?

For that matter, should FCP be a "status"?  Should we have a separate tracker for RFC statuses, maybe a table in the README or a separate top-level file?

Reply:
I like the timeline idea. I'll add that in a new commit shortly.

Another possibility is to use GitHub labels. For example, we could label PRs with `pending`, `final-comment-period`, `active`, `landed`, or `rejected`. My only hesitation is that labels are easy to (intentionally or unintentionally) change whereas changing content in a file requires an explicit commit.

Reply:
Updated the PR, see what you think.

@cdisselkoen I will be AFK from now until Monday, so feel free to take over this PR in the meantime. Otherwise I'll get back to it in a couple days.